osu.Framework/Extensions/Color4Extensions/Color4Extensions.cs

```diff
@@ -11,26 +11,34 @@ public static class Color4Extensions
     {
         public const double GAMMA = 2.4;
 
-        public static double ToLinear(double color) => color <= 0.04045 ? color / 12.92 : Math.Pow((color + 0.055) / 1.055, GAMMA);
+        // ToLinear is quite a hot path in the game.
+        // MathF.Pow performs way faster than Math.Pow, how ever on Windows it lacks a short path for x == 1
+        // Given passing color == 1 (White or Transparent) is very common, a short path for 1 is added.
```

Comment:
```suggestion
        // MathF.Pow performs way faster than Math.Pow, however on Windows it lacks a short path for x == 1
        // Given passing color == 1 (White or Transparent) is very common, a short path for 1 is added.
```

Also I believe the widely-accepted term is [_fast path_](https://en.wikipedia.org/wiki/Fast_path) rather than _short path_. But that's mostly beside the point.